src/Sulu/Bundle/MediaBundle/DependencyInjection/ImageFormatCompilerPass.php

```diff
@@ -47,41 +47,54 @@ public function process(ContainerBuilder $container)
     protected function loadThemeFormats($defaultOptions)
     {
         $activeFormats = array();
-        $suluFormats = $this->container->getParameter('sulu_media.image.formats');
-        if (is_array($suluFormats)) {
-            $activeFormats = $suluFormats;
-        }
+        $this->setFormatsFromFile(__DIR__ . '/../Resources/config/image-formats.xml', $activeFormats, $defaultOptions);
 
         $activeTheme = $this->container->get('liip_theme.active_theme');
         $bundles = $this->container->getParameter('kernel.bundles');
+        $configPaths = $this->container->getParameter('sulu_media.format_manager.config_files');
```

Comment:
Shouldn't the parameter be called `config_paths`? It's not pointing to a direct file, is it? At least the variables should have the same name.